### DISCUSSION TITLE
Have uninstalled Immich but images still on computer when reinstall immich. How erase them

### DISCUSSION THREAD

**Discussion by Havouza15** [immich-app/immich#7883] [Q&A] [answered]
I want to totally uninstall immich to start over but when I have deleted all containers and reinstall immich the images show up directly. How erase them before reinstall

**Comment by MozPri** [comment]
docker image prune

https://docs.docker.com/reference/cli/docker/image/prune/

**Comment by bo0tzz** [comment, marked as answer]
https://immich.app/docs/FAQ/#how-can-i-purge-data-from-immich
